### DISCUSSION TITLE
How to downgrade to a specific patch version

### DISCUSSION THREAD

**Discussion by honzavaclavik** [shivammathur/homebrew-php#1720] [Q&A / Help] [answered]
Hello, 

I'm wondering how downgrade PHP8.1.19 to a specific patch version eg. 8.1.16.

I'm trying to solve an issue on my projects and it seems to be related to 8.1.19 patch version.

Thanks in advance.

**Comment by shivammathur** [comment, marked as answer]
@honzavaclavik 
Please refer to the answer here
https://stackoverflow.com/questions/75835275/how-to-install-specific-older-php-8-1-version-on-mac-using-brew/75848252#75848252

**Reply by honzavaclavik** [reply]
Thanks
